### PR TITLE
Apply PHPCS 2.x autofixers

### PR DESCRIPTION
**Pull request by photodude**
apply some autofixers

### CODE REVIEW

**Comment by photodude** [comment]
@wilsonge I applied the PHPCS 2.x autofixers to some of the files in your PR. Feel free to accept or not.

File: tests/unit/suites/libraries/cms/router/JRouterTest.php [hunk 1]
```diff
@@ -132,7 +132,8 @@ public function testLegacyApplicationRouterIsStillLoaded()
 			'id'   => 3,
 			'name' => 'tester',
 			'path' => __DIR__ . '/data'
-		));
+			)
```
Comment (wilsonge) [inline tests/unit/suites/libraries/cms/router/JRouterTest.php:135]: This seems a bit weird?

**Comment by wilsonge** [comment]
This looks really good :) The array close tag seems a bit odd but otherwise seems fine :)

File: tests/unit/suites/libraries/cms/router/JRouterTest.php (hunk 1 above)
Comment (photodude) [reply tests/unit/suites/libraries/cms/router/JRouterTest.php:135]: So technically this is "Closing parenthesis of a multi-line function call must be on a line by itself" which is the PEAR.Functions.FunctionCallSignature.CloseBracketLine rule. I somewhat agree that it looks weird but that is the rule our coding standard applies. It's the same rule in phpcs 2.x as the old phpcs 1.5.x custom sniff [Joomla.Functions.FunctionCallSignature.CloseBracketLine](https://github.com/joomla/coding-standards/blob/master/Sniffs/Functions/FunctionCallSignatureSniff.php#L249)

**Pull request merged**